lib/instrumental_tools/system_inspector/linux.rb

```diff
@@ -96,6 +97,11 @@ def self.memory
 
       }
 
+      if available
+        stats_to_record['memory.available_mb']      = available / 1024
+        stats_to_record['memory.available_percent'] = (available / total) * 100
```

Comment:
`available` has already been `to_f`'ed when it is set above, so oversight or am I missing something?